docs/E-Secondary-Storage/records-and-files.md

```diff
@@ -19,9 +22,9 @@ This chapter describes how to identify records and fields in a text file and how
 
 A **record** occupies a single line in a text file and holds all of the data associated with one chunk of information.  The record is a sequence of characters that ends with a record delimiter.  The typical record delimiter is the newline character (`\n`). 
 
-~[](https://ict.senecacollege.ca//~ipc144/pages/images/records.png)
+![Each records end with a record delimiter.](https://ict.senecacollege.ca//~ipc144/pages/images/records.png)
```

Comment:
"Each record ends..."

Also, you have to bring that image into this repo vs. using the external link.

Reply:
ok. I will do the change.